src/base/ShapesFactory.js

```diff
@@ -0,0 +1,33 @@
+/**
```

Comment:
Please remove all the Adobe copyright headers.

Reply:
ok